app/build.gradle

```diff
@@ -62,6 +63,10 @@ dependencies {
 
     testCompile 'junit:junit:4.12'
 
+    compile "com.google.firebase:firebase-auth:$firebaseVersion"
+    compile "com.google.android.gms:play-services-auth:$firebaseVersion"
+    compile 'com.facebook.android:facebook-android-sdk:4.18.0'
```

Comment:
Can we upgrade to newest version ? More in changelog https://developers.facebook.com/docs/android/change-log-4x